src/administration/troubleshooting.md

```diff
@@ -64,7 +64,25 @@ curl -H "Accept: application/activity+json" https://your-instance.com/comment/12
 
 Check that [federation is allowed on both instances](federation_getting_started.md).
 
-Also ensure that the time is accurately set on your server. Activities are signed with a timestamp, and will be discarded if it is off by more than 10 seconds.
+Also ensure that the time is correct on your server. Activities are signed with a timestamp, and will be discarded if it is off by more than one hour.
+
+It is possible that federation requests to `/inbox` are blocked by tools such as Cloudflare. The sending instance can find HTTP errors with the following steps:
+
+- Set `RUST_LOG=lemmy_federate=trace` for Lemmy
+- Reload the new configuration: `docker compose up -d`
+- Search for messages containing the target instance domain: `docker compose logs -f --tail=100 lemmy-federate |  grep lemm.ee -C 10`
+- If you use a [separate container for outgoing federation](horizontal_scaling.md), you need to apply the previous steps to that container only
```

Comment:
this should probably be moved to the beginning of the list

Reply:
Changed this one too, thanks for the suggestions!